rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2020-05-15
+nightly-2020-08-26
```

Comment:
this is very unexpected to do this in this PR

Reply:
Yeah i'm aware of this. Using this only because 05-15's rust language server is broken on linux and i had to use a working one to speed up coding. Will undo this after i'm done